### PR TITLE
New planimtries + Changed logic for side groups, description always visibile, removed more info link + Changed logos and favicon for work-container (b&w version)

### PR DESCRIPTION
**Pull request by madeincima**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+	OPEN_DATA_HUB_TYPES: 'https://mobility.api.opendatahub.bz.it/v2/flat/NOI-Place/*/latest?limit=-1&offset=0&select=tmetadata,mvalue&shownull=false&distinct=true&where=mvalue.eq.Selettori%20Type',
+	OPEN_DATA_HUB_TYPES_GROUPS: 'https://mobility.api.opendatahub.bz.it/v2/flat/NOI-Place/*/latest?limit=-1&offset=0&select=tmetadata,mvalue&shownull=false&distinct=true&where=mvalue.eq.Selettori%20Group',
+	OPEN_DATA_HUB_BUILDINGS: 'https://mobility.api.opendatahub.bz.it/v2/flat/NOI-Place?limit=-1&offset=0&select=sname,smetadata&where=smetadata.building_code.neq.null&where=smetadata.building_name.neq.null',
+	OPEN_DATA_HUB_FLOORS: 'https://mobility.api.opendatahub.bz.it/v2/flat/NOI-Place?limit=-1&offset=0&select=sname,smetadata&where=smetadata.floor_name.neq.null&where=smetadata.image.neq.null&where=smetadata.show_on_map.eq.1',
+	OPEN_DATA_HUB_TRANSLATIONS: 'https://mobility.api.opendatahub.bz.it/v2/flat/NOI-Place/*/latest?limit=-1&offset=0&select=tmetadata,mvalue&shownull=false&distinct=true&where=mvalue.eq.Traduzioni',
```
Comment (Piiit) [inline src/config.js:9]: @madeincima Please keep the configurable endpoint, we need it for testing...

**Review by Piiit — CHANGES_REQUESTED** [review]

**Comment by Piiit** [comment]
@madeincima Please resolve the conflicts... thank you

**Comment by madeincima** [comment]
Please check the correct syntax in **config.js** (we haven't yet seen process.env.ODH_BASE_URL with office365 and the added a new resource url "OPEN_DATA_HUB_TYPES" and changed "OPEN_DATA_HUB_TYPES_GROUP") and the compiled **noi_maps_widget.min.js** using the new config.js (we cannot test using process.env.ODH_BASE_URL)

**Comment by Piiit** [comment]
@madeincima Thx for reporting... I will have a look at the config.js file and try to solve it... do you have an idea, how we could fix this, such that it is still configurable via env vars, but works also in local development?

**Comment by Piiit** [comment]
@madeincima Please start your future PRs from the development branch... thx

**Comment by Piiit** [comment]
@madeincima I tried several ways to solve the config.js problem, but the preview served by the webpack dev server always reloads continuously, except I remove the process.env constants from the config.js file, so I will in the meantime just merge this PR and use your previous solution on top of it to hardcode the endpoints. I will also open a new issue to solve the local development with env vars. Thx

**Comment by Piiit** [comment]
@madeincima I updated the repository on development... I removed the write-to-disk during webpack development stages, cleared some whitespace issues and added a method to configure/replace image urls... all of these things take the .env file to configure stuff... please use that as well... see .env.example, copy that to .env and put your endpoints... "npm run start" resolves that then for you, write me if you have questions.

I have seen that many times on error a reload() gets fired, that makes it impossible to debug code... on every error it gets into an endless loop of reloading... could you please solve those occasions?

**Pull request merged**